app/controllers/publishers_controller.rb

```diff
@@ -189,9 +190,9 @@ def new_auth_token
   end
 
   def create_auth_token
-    email = publisher_create_auth_token_params[:email]
+    @publisher_email = publisher_create_auth_token_params[:email]
```

Comment:
`@publisher_email` used in emailed_auth_token.rb can be `publisher.email` or `publisher.pending_email` so it makes sense to set this here